src/wallet/wallet.cpp

```diff
@@ -8152,7 +8156,10 @@ bool CWallet::InitLoadWallet() {
         LogPrintf("Rescanning last %i blocks (from block %i)...\n", chainActive.Height() - pindexRescan->nHeight,
                   pindexRescan->nHeight);
         nStart = GetTimeMillis();
-        walletInstance->ScanForWalletTransactions(pindexRescan, true);
+        bool fRecoverMnemonic = false;
+        if (GetBoolArg("-fRecoverMnemonic", false))
```

Comment:
all the other possible command line arguments look in specific way, this has to be renamed